tests/integration-tests/configs/develop.yaml

```diff
@@ -361,15 +361,15 @@ test-suites:
         # S3 bucket belonging to the same region and S3 VPC Endpoints only work within the region.
         - regions: ["us-east-1"]
           instances: {{ common.INSTANCES_DEFAULT_ARM }}
-          oss: [{{ OS_ARM_3 }}]
+          oss: [{{ DCV_OS_ARM_3 }}]
           schedulers: ["slurm"]
         - regions: ["cn-north-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: [{{ OS_X86_5 }}]
+          oss: [{{ DCV_OS_X86_3 }}]
           schedulers: ["slurm"]
         - regions: ["us-gov-west-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: [{{ OS_X86_7 }}]
+          oss: [{{ DCV_OS_X86_4 }}]
```

Comment:
Minor: It is better to use `DCV_OS_X86_5`, so that the test is running on 4 OSes in consecutive two days